### PR TITLE
blog: add new post "first steps towards urbit.org", css: small footnote styling edits

### PR DESCRIPTION
**Pull request by urcades**
Final draft of blog post added, some minor footnote restyling added.

### CODE REVIEW

**Comment by urcades** [comment]
Ah, I need to add a description in the metadata. Will fix.

**Comment by crrncllns** [comment]
looks great

File: content/blog/first-steps-towards-urbit-org.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
++++
+title = "First Steps Towards urbit.org"
+date = 2020-08-27
```
Comment (matildepark) [inline content/blog/first-steps-towards-urbit-org.md:3]: Fifteen days from now ...

File: content/blog/first-steps-towards-urbit-org.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+The product work that Tlon is doing is crucial to Urbit’s long-term success, so I’ll also be leveraging bounties to more directly accelerate their development efforts. Landscape represents the clearest example of a cohesive vision for Urbit’s use. The existence of a polished product built on Urbit demonstrates to the world that the platform is robust and capable, and the growing community seems eager to help. You can expect more bounties very soon that provide a direct connection to the Landscape roadmap.
+
+As I’m only just getting started, I’d love to hear your thoughts. If you’re working on any grants you’ll hear from me soon, and if you have an idea that you want to pursue, I’m here to help you make it happen. You can reach me directly via Urbit as ~wolref-podlex or by email at josh@urbit.org.
```
Comment (matildepark) [inline content/blog/first-steps-towards-urbit-org.md:51]: Backtics on the patp. Maybe the email too, I can't recall if we do that.

File: content/blog/first-steps-towards-urbit-org.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
++++
+title = "First Steps Towards urbit.org"
+date = 2020-08-27
+description = "With a stable platform taking shape and a strong community forming that wants to help build Urbit, it’s time to make urbit.org real. We’ve had plenty of time to think about what it should look like, how it should work, and who should lead it."
```
Comment (matildepark) [inline content/blog/first-steps-towards-urbit-org.md:4]: Really should just be the first sentence here...

File: sass/_main.scss [hunk 4]
```diff
@@ -216,16 +216,17 @@ body.essay {
         display: block;
     }
     div.footnote-definition {
-      &[id="0"] {
+      &[id="1"] {
         margin-top: 4rem;
-        padding-top: 2rem;
+        padding-top: 4rem;
         border-top: 1px solid black;
       }
```
Comment (matildepark) [inline sass/_main.scss:223]: That produces this, which I can't think is intentional:

image

**Comment by urcades** [comment]
Thanks for the edits/eyes @matildepark — just fixed via master branch

**Pull request merged**